cts/cts-cli.in

```diff
@@ -119,15 +123,35 @@ function _test_assert() {
     printf "* Running: $app - $desc\n" 1>&2
 
     printf "=#=#=#= Begin test: $desc =#=#=#=\n"
-    eval $VALGRIND_CMD $cmd 2>&1
+
+    export outfile=$(mktemp ${TMPDIR:-/tmp}/cts-cli.output.XXXXXXXXXX)
+    eval $cmd 2>&1 | tee $outfile
```

Comment:
Did you mean to kill valgrind here?

Reply:
Nope, that must have been an artifact of when I was first working on the patch.  Fixed.